src/main/java/ng/abdlquadri/eventbus/EventBus.java

```diff
@@ -232,17 +244,21 @@ public static void close() {
       closeFuture.addListener(new ChannelFutureListener() {
         public void operationComplete(ChannelFuture future) throws Exception {
           if (future.isSuccess() && future.isDone()){
-            log.log(Level.SEVERE, "Channel Successful Closed");
+           // silent exit is better
+           //  log.log(Level.SEVERE, "Channel Successful Closed");
 
           }else {
-            log.log(Level.SEVERE, "Channel Failed Closed {1}", future.cause());
+           // silent exit is better
+           //   log.log(Level.SEVERE, "Channel Failed Closed {1}", future.cause());
 
           }
         }
       });
     } else {
-      log.log(Level.SEVERE, "Channel is not connected. You can not close a non existent connection :). Make sure the server is reachable and call EventBus.connect() method first.");
-      throw new IllegalStateException("Channel is not connected. You can not close a non existent connection :). Make sure the server is reachable and call EventBus.connect() method first.");
+      // silent exit is better 
+      // log OR throw exception, never both
+      // log.log(Level.SEVERE, "Channel is not connected. You can not close a non existent connection :). Make sure the server is reachable and call EventBus.connect() method first.");
+      // throw new IllegalStateException("Channel is not connected. You can not close a non existent connection :). Make sure the server is reachable and call EventBus.connect() method first.");
```

Comment:
Hi @nielsbaloe , you will have to make the change and ensure it works well for you. It's your PR <3